.github/workflows/CI.yml

```diff
@@ -18,6 +18,14 @@ jobs:
         with:
           cache: npm
           node-version: ${{ matrix.node-version }}
+      - name: Find dependency duplications
+        run: npm dedupe
+      - name: Check differences
+        run: git status -s
+          | grep " M package-lock.json"
+          && echo "warning file=package-lock.json Duplicate dependencies found. Run npm dedupe to remove them, or add --dry-run parameter to get more info."
+      - name: Reset changes
+        run: git reset --hard HEAD
```

Comment:
1. I'd rather do it in a single step.
2. Can we add a similar one for `npm audit fix`?